modules/central-services-private-config.adoc

```diff
@@ -93,10 +93,10 @@ For setting the administrator password, you can only use either `central.adminPa
 | A private key which {product-title} should use for signing JSON web tokens (JWTs) for authentication.
 
 | `central.serviceTLS.cert`
-| An internal certificate that the `central.stackrox` service should use for deploying Central.
+| An internal certificate that the Central service should use for deploying Central.
 
 | `central.serviceTLS.key`
-| The internal certificate key that the `central.stackrox` service should use.
+| The internal certificate key that the Central service should use.
```

Comment:
nit: Is there such a thing as "certificate key?"  I've only read of public keys and private keys.  If I knew anything about this, I'd actually be of use to you.  I guess no customer is confused.

Reply:
Thanks for this one, I've fixed it.